src/i18n/en-US/discussions.ts

```diff
@@ -0,0 +1,80 @@
+const discussions = {
+  // TODO: We need to make a decision on how to structure these translations:
+  //         - Do we want a generic i18n file at all?
+  //         - Should some board specific stuff be stored in their respective i18n files? (e.g. name of the board, participants, role type translations, etc. in grbReview.ts)
```

Comment:
I think higher level keys in this file for more specific contexts is good enough